<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QWidget" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messenger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="40,100">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="createGroup">
       <property name="styleSheet">
        <string notr="true">*:hover {
  background-color: rgb(98,194,233);
}</string>
       </property>
       <property name="text">
        <string>create group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectGroup">
       <property name="styleSheet">
        <string notr="true">*:hover {
  background-color: rgb(98,194,233);
}</string>
       </property>
       <property name="text">
        <string>connect group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="groups">
       <property name="styleSheet">
        <string notr="true">* { 
  font: bold;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="users"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
     <item>
      <widget class="QListView" name="chatView">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="messageEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">* {
  font: 15px;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>30</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">* {
  border-image: url(:/sendButton.png) 3 10 3 10;
  border-top: 3px transparent;
  border-bottom: 3px transparent;
  border-right: 10px transparent;
  border-left: 10px transparent;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
